lib/index.js

```diff
@@ -45,13 +45,28 @@ Seal.prototype.move = function() {
   return this;
 }
 
+Seal.prototype.detectCollision = function(){
+  if (this.x > canvas.width){
+    this.speed = -1*this.speed;
+  } else if (this.x < 0){
+    this.speed = -1*this.speed;
+  }
+}
+
```

Comment:
Move prototype switched over to detectCollision. Seals will now reverse direction when they reach the limits of the canvas